docs/openapi/components/schemas/common/AgGradeInspectionSampleProperty.yml

```diff
@@ -0,0 +1,38 @@
+$linkedData:
+  term: AgGradeInspectionSampleProperty
+  '@id': https://w3id.org/traceability#AgGradeInspectionSampleProperty
+title: Agriculture Grade Inspection Sample Property
+description: A property of some agriculture grade inspection sample.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - AgGradeInspectionSampleProperty
+      - type: string
+        const:
+          - AgGradeInspectionSampleProperty
+  propertyName:
+    title: Property Name
+    description: Name of the sample property observed (e.g. "Percent off-size", "Type of Defects", "Net Wt. Within 2 lb. Range").
+    type: string
+    $linkedData:
+      term: propertyName
+      '@id': https://schema.org/name
+  propertyValue:
+    title: Property Value
+    description: Value of the sample property observed (e.g. "0", "SIL CT", "Yes").
+    type: string
+    $linkedData:
+      term: propertValue
+      '@id': https://schema.org/description
+additionalProperties: false
+example: |-
+  {
+    "type": "AgGradeInspectionSampleProperty",
+    "propertyName": "Temp °F",
+    "propertyValue": "53"
```

Comment:
do you want this to be a number or string?

Reply:
This is a bit kludgey but I did intend propertyValue to be a string, even if that string sometimes contains only a number. My thinking was that sample property values really just need to be discernible by humans, and that setting it up to contain either a number or string as appropriate for the unit would result in a convoluted form for inspectors filling it out.

I'm familiar with the use of MeasuredValue & MeasuredProperty, but I don't think those fit super well and IMO would bloat the form.

Reply:
https://v.jsld.org/Vjxdn8aYZeKGBe7fe798UpNutDH4JXy9NXbxeGXWNthtNLd4mKZz5SH3mjQsZA5hfe1HotykArBF16gDcCWFvhqXCaAgxXrVEACXRzwWEAY3tZghdvJKmwS4pXomkjU9hDhQsZbtqxudcW6hdNbweTVJaDpq722ykkuYFQgTgNav3R8SCxC7MN3WmwL4yvDNqxA5e1qZmphkzDkdyNP4cjKgDGLry9yRrAb8WdB9PuZtc76WbugvjeM8LkVeqU

```
_:c14n0 <https://w3id.org/traceability/#undefinedTermentityType> "Organization" .
```